packages/web-react/src/components/Dropdown/README.md

```diff
@@ -52,20 +52,43 @@ import { UncontrolledDropdown, DropdownTrigger, DropdownPopover } from '@lmc-eu/
 
 ### Dropdown
 
-| Name              | Type                                               | Default        | Required | Description                                    |
-| ----------------- | -------------------------------------------------- | -------------- | -------- | ---------------------------------------------- |
-| `enableAutoClose` | `bool`                                             | `true`         | ✕        | Enables close on click outside of Dropdown     |
-| `fullWidthMode`   | [`DropdownFullWidthMode`][dropdown-fullwidth-mode] | `off`          | ✕        | Full-width mode                                |
-| `id`              | `string`                                           | —              | ✓        | Component id                                   |
-| `isOpen`          | `bool`                                             | `false`        | ✓        | Open state                                     |
-| `onAutoClose`     | `(event: Event) => void`                           | —              | ✕        | Callback on close on click outside of Dropdown |
-| `onToggle`        | `() => void`                                       | —              | ✓        | Function for toggle open state of dropdown     |
-| `placement`       | [Placement dictionary][dictionary-placement]       | `bottom-start` | ✕        | Alignment of the component                     |
+| Name              | Type                                                                  | Default        | Required | Description                                                                                                                               |
+| ----------------- | --------------------------------------------------------------------- | -------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `alignmentX`      | \[ [AlignmentXExtended dictionary][dictionary-alignment] \| `object`] | `null`         | ✕        | Apply vertical alignment to trigger, use object to set responsive values, e.g. `{ mobile: 'left', tablet: 'center', desktop: 'right' }`   |
+| `alignmentY`      | \[ [AlignmentYExtended dictionary][dictionary-alignment] \| `object`] | `null`         | ✕        | Apply horizontal alignment to trigger, use object to set responsive values, e.g. `{ mobile: 'top', tablet: 'center', desktop: 'bottom' }` |
```

Comment:
Are the props intentionally nullable? Do we have it like this in other components with alignment?

Reply:
They are. Because you don't need to set them. If you don't set them the class sets only position. If you set anything it is display flex as well (it would probably be a BC if it was flex always).